modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/base_controller.rb

```diff
@@ -167,6 +169,24 @@ def nullable_icn
 
           nil
         end
+
+        def fetch_claimant
+          claimant_icn = form_attributes.dig('claimant', 'claimantId')
+          if claimant_icn.present?
+            user_profile = mpi_service.find_profile_by_identifier(identifier: claimant_icn,
+                                                                  identifier_type: MPI::Constants::ICN)
+          end
+        rescue ArgumentError
```

Comment:
Is the `ArgumentError` coming from `mpi_service`?

Reply:
Yes, that's returned in cases where MPI can't find a user profile for the specified ICN.